### DISCUSSION TITLE
unsupported RTSP dialect

### DISCUSSION THREAD

**Discussion by mohankumar-dev-2002** [bluenviron/mediamtx#3021] [Questions]
### Question

I have configured the camera in mediamtx.yml and while starting the server it's working fine but while requesting for RTSP
 stream from the server(its happening often) I'm getting  closed: path of a SETUP request must end with a slash. This typically happens when VLC fails a request, and then switches to an unsupported RTSP dialect.
 
![Screenshot from 2024-02-14 17-02-45](https://github.com/bluenviron/mediamtx/assets/75905470/34431f1b-4778-4bc7-a65c-cf0d64844cd6)

**Comment by aler9** [comment]
As written in logs, you are publishing a stream to path "/camara1", at the same time you are trying to read a stream to path "/camera1", which of course doesn't exist.

[a comment by an automated account is not shown]
